microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -243,10 +243,26 @@ Collecting child browsing data and sharing it with the family group organizer re
 The family group organizer may stop the data collection from the family safety portal.  For more information about Microsoft family safety features, navigate to [What is a Microsoft family group?](https://support.microsoft.com/help/12413)
 
 
+<!-- ====================================================================== -->
+## Find on Page 
+
+Find on Page allows you to search a web page for important keywords. Open **Settings and more (...)** > **Find on Page**. A search box will appear. Type any keyword or phrase into this search box and the page will highlight all locations on the page where your exact search term appears. You can then navigate through these terms with the **Previous result** and **Next result** buttons. 
```

Comment:
```suggestion
**Find on Page** allows you to search a web page for important keywords.  Select **Settings and more** (**...**)<!--(!['Settings and more' icon.](media/edge-settings-icon.png))--> > **Find on Page** (`Ctrl`+`F`).   A search box appears.  Type any keyword or phrase into this search box, and the page will highlight all locations on the page where your exact search term appears.  You can then navigate through these terms with the **Previous result** and **Next result** buttons.
```

Reply:
We use "open Settings and more (...)" in 7 other locations in the privacy whitepaper. We do not have any references to hot key controls today.